mesos-cluster.tf

```diff
@@ -30,3 +30,8 @@ module "mesos-cluster-production" {
     subnet2              = "${aws_subnet.apps-production-1c.id}"
     subnet3              = "${aws_subnet.apps-production-1d.id}"
 }
+
+module "mesos-cluster-cloudwatch" {
```

Comment:
This isn't quite right. We'd need to add a line to the existing modules to set a variable for them. This would try to create a new mesos cluster using the mesos-cluster module.

We'd need to add `sns_topic_arn = ${aws_sns_topic.sns-cloudwatch-partinfra.arn}` to the mesos-cluster-staging and mesos-cluster-production resources